snuba/query/allocation_policies/bytes_scanned_rejecting_policy.py

```diff
@@ -314,3 +311,71 @@ def _update_quota_balance(
             ],
             timestamp=int(time.time()),
         )
+
+    def get_throttle_threshold(self, tenant_ids: dict[str, str | int]) -> int:
+        return max(
+            1,
+            typing.cast(
+                int,
+                self.get_rejection_threshold(tenant_ids)
+                // self.get_config_value("bytes_throttle_divider"),
+            ),
+        )
+
+    def get_rejection_threshold(self, tenant_ids: dict[str, str | int]) -> int:
+        (
+            customer_tenant_key,
+            customer_tenant_value,
+        ) = self._get_customer_tenant_key_and_value(tenant_ids)
+        referrer = tenant_ids.get("referrer", "no_referrer")
+        if referrer in _PASS_THROUGH_REFERRERS:
+            return sys.maxsize
+        return self.__get_scan_limit(
+            customer_tenant_key, customer_tenant_value, referrer
+        )
+
+    def get_quota_used(self, tenant_ids: dict[str, str | int]) -> int:
```

Comment:
@volokluev I'm not a fan of repeating this chunk of code

Reply:
so why are you doing it then?